app/templates/components/cells/schema-name-entry.hbs

```diff
@@ -9,9 +9,9 @@
 {{#if row.hasEnumerations}}
   <span class="schema-enumeration-caret">
     {{#if row.expanded}}
-      <i class="expanded-caret glyphicon glyphicon-menu-down"></i>
+      <i class="expanded-caret fa fa-chevron-down"></i>
```

Comment:
change to {{ fa-icon down class="expanded-caret"}}?

Reply:
Done